source/CardGames.py

```diff
@@ -190,8 +190,22 @@ def __init__(self):
         self.fugitive = Player("", "Fugitive")
         self.marshall = Player("", "Marshall")
         self.done = False
+
+    def ReadRules(self):
+        choice = ""
+        while choice != "1":
```

Comment:
It might be better to use an OR statement such as choice != "1" or choice != "0" to get rid of the if statement within the while loop.

Reply:
Thanks, Ian! I am not sure why I used an if statement instead of an or statement in the first place. I will be sure to change it.

Reply:
Thanks, Ian! I am not sure why I used an if statement instead of an and statement in the first place. Also, I believe it should be and instead of or since it would go in an infinite loop if it was or.  I will be sure to change it.